libs/api/src/lib/media/novaMediaAdapter.ts

```diff
@@ -146,12 +155,35 @@ export class NovaMediaAdapter implements MediaAdapter {
     if (transformations?.sharpen) {
       queryParameters.push(`sharpen=1`)
     }
+    **/
+    let quality = transformations?.quality
+    if (quality) {
+      quality = sanitizeImageQuality(quality)
+    }
 
     // Max quality is 80 so 1 => 80
-    queryParameters.push(
-      `quality=${transformations?.quality ? Math.ceil(transformations.quality * 80) : 65}`
-    )
+    queryParameters.push(`quality=${quality ? Math.ceil(quality * 80) : 65}`)
+
+    const dto = this.parseTransformations(queryParameters)
```

Comment:
_dto_ as a const name sounds quite cryptic to me.

Reply:
Changed name